iohk-monitoring/src/Cardano/BM/Configuration.lhs

```diff
@@ -66,7 +66,7 @@ testSeverity :: CM.Configuration -> LoggerName -> LOMeta -> IO Bool
 testSeverity config loggername meta = do
     globminsev  <- CM.minSeverity config
     globnamesev <- CM.inspectSeverity config loggername
-    let minsev = globminsev `max` fromMaybe Debug globnamesev
+    let minsev = globminsev <> fromMaybe Debug globnamesev
```

Comment:
```suggestion
    let minsev = globminsev <> fold globnamesev
```
The above line is wrong.

Reply:
Right, yes!

Got too carried away with the min/max semantics and writing an explanation.. : -)